### DISCUSSION TITLE
Error DreamBooth

### DISCUSSION THREAD

**Discussion by Alexssd22** [TheLastBen/fast-stable-diffusion#1689] [General]
I have this problem, I can't use an existing template. Does anyone have a solution?

(link https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast-DreamBooth.ipynb#scrollTo=qEsNHTtVlbkV)

Converting to Diffusers ...
Traceback (most recent call last):
  File "/content/convertodiff.py", line 1130, in <module>
    convert(args)
  File "/content/convertodiff.py", line 1081, in convert
    text_encoder, vae, unet = load_models_from_stable_diffusion_checkpoint(v2_model, args.model_to_load)
  File "/content/convertodiff.py", line 846, in load_models_from_stable_diffusion_checkpoint
    info = unet.load_state_dict(converted_unet_checkpoint)
  File "/usr/local/lib/python3.8/dist-packages/torch/nn/modules/module.py", line 1671, in load_state_dict
    raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for UNet2DConditionModel:
	Missing key(s) in state_dict: "up_blocks.0.upsamplers.0.conv.weight", "up_blocks.0.upsamplers.0.conv.bias", "up_blocks.1.upsamplers.0.conv.weight", "up_blocks.1.upsamplers.0.conv.bias", "up_blocks.2.upsamplers.0.conv.weight", "up_blocks.2.upsamplers.0.conv.bias". 
	Unexpected key(s) in state_dict: "up_blocks.0.attentions.2.conv.bias", "up_blocks.0.attentions.2.conv.weight". 
rm: cannot remove '/content/stable-diffusion-custom': No such file or directory
Conversion error
Conversion error

**Comment by TheLastBen** [comment]
does the custom model have a diffusers version in huggingface ?

**Reply by Alexssd22** [reply]
I tested from huggingface, google drive and CiviAI and none worked.
Do you have any models that work there?

**Reply by TheLastBen** [reply]
merged models tend to not convert correctly, if the model has the diffusers version (multiple folders), use that one instead of the ckpt

**Reply by Alexssd22** [reply]
I couldn't use any templates to convert unfortunately

**Reply by THX1139b** [reply]
I'm having the same problem. I get conversion error on the included SDXL model. I mean the one that is included in the auto models folder. I can load the 1.5 model but I cannot get any XL model to work. But I guess there is a way to do it since it is named RunPod(SDXL Trainer). 

I would love it if anyone knows of a guide to all the steps that make this work.

**Reply by TheLastBen** [reply]
@THX1139b there is SDXL LoRA trainer notebook designed to train SDXL LoRA

**Reply by THX1139b** [reply]
Aha. Thanks. I thought it was possible to train full models for SDXL with this. Haven't had much success with Loras for likeness.
